### PR TITLE
Uses same seed to initialize each experimental trial.

### PR DESCRIPTION
**Pull request by matthewclegg**
The purpose of this change is to help ensure that different modeling strategies are compared using exactly the same inputs.

### CODE REVIEW

File: src/driver/experimental_trial.py [hunk 1]
```diff
@@ -114,6 +114,9 @@ def evaluate(self, rng: np.random.Generator) -> pd.DataFrame:
         logging.vlog(2, f"Dataset {self._data_set_name}")
         logging.vlog(2, f"Trial   {self._trial_descriptor}")
 
+        rng = np.random.default_rng(seed=seed)
+        np.random.seed(seed)
```
Comment (jiayu-google) [inline src/driver/experimental_trial.py:118]: Could you explain the function of this line in addition to the above line?

**Review by jiayu-google — APPROVED** [review]
Thanks for the hard work, Matt!

File: src/driver/experimental_trial.py (hunk 1 above)
Comment (matthewclegg) [reply src/driver/experimental_trial.py:118]: In earlier distributions of numpy, random numbers were generated via RandomState.  This was subsequently replaced with Random Generator.  The Planning Evaluation Framework uses the more modern Random Generator throughout.  The seed for this generator is initialized at line 117.  However, some of the code that we depend on (specifically, PyDOE.lhs) uses the legacy RandomState.  The seed for this generator is initialized at line 118.

For more information, see the following:
https://numpy.org/doc/stable/reference/random/generator.html

**Review by RonPinelliMRC — APPROVED** [review]
MRC approving based on cursory review after confirming non-independent code review on an exception basis. Experimental code will require independent review prior to publication of or reliance on results generated.

**Pull request merged**